web-frontend/modules/core/assets/scss/components/radio_card.scss

```diff
@@ -0,0 +1,40 @@
+.radio-card {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding: 14px 16px 16px;
+  border-radius: 6px;
+  border: 1.5px solid $palette-neutral-400;
+  box-shadow: 0 1px 2px 0 rgba(7, 8, 16, 0.1);
+}
+
+.radio-card--selected {
+  border: 1.5px solid #4e5cfe;
```

Comment:
This is not something you could have known, but would you mind replacing this color with `$palette-blue-500`?